openwisp_notifications/templates/admin/change_form.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'admin/change_form.html' %}
+
+{% block admin_change_form_document_ready %}
+    {{ block.super }}
+    <script>
+        const owNotifyAppLabel = "{{ opts.app_label }}",
+            owNotifyModelName = "{{ opts.model_name }}",
```

Comment:
here you're using double quotes but single quotes below, let's keep consistency and use single quotes